server/routes/volunteers.ts

```diff
@@ -177,13 +181,13 @@ router.route("/api/volunteers/update/:volunteerId").patch(
       lastName: string;
       email: string;
       phoneNumber: string;
-      participantType: string[];
+      participantType: ParticipantType[];
     } = req.body;
 
     const stringFields = [firstName, lastName, email, phoneNumber];
 
     const isValidRequest = stringFields.every((field) => {
-      return field.trim().length > 0;
+      return field.trim().length > 0 && typeof field === "string";
```

Comment:
This should be flipped, since we need to check first that it's of "string" type, and then check that it's non-empty. Otherwise, `field.trim()` will throw an exception.

Reply:
Good catch, did the same for checking each volunteer type also